src/render_tasks/d3d12_equirect_to_cubemap.hpp

```diff
@@ -180,7 +180,7 @@ namespace wr
 
 			data.in_equirect = skybox_node->m_hdr;
 			
-			skybox_node->m_skybox = skybox_node->m_hdr.m_pool->CreateCubemap("Skybox", 3840, 3840, 0, wr::Format::R32G32B32A32_FLOAT, true);
+			skybox_node->m_skybox = skybox_node->m_hdr.m_pool->CreateCubemap("Skybox", 1024, 1024, 0, wr::Format::R32G32B32A32_FLOAT, true);
```

Comment:
Make this a d3d12_settings variable plz :)

Reply:
Done